baseframe/static/js/baseframe-material.js

```diff
@@ -249,6 +249,45 @@ window.Baseframe.Forms = {
         }
       }
     })
+  },
+  /* Takes 'formId' and 'errors'
+     'formId' is the id attribute of the form for which errors needs to be displayed
+     'errors' is the wtf form validation errors expected in the following format
```

Comment:
Typo: it's `WTForm` or `wtform`, not `wtf form`.

Reply:
Fixed